specfile/rpm.py

```diff
@@ -352,6 +352,7 @@ def get_included_sources(content):
             # do a non-build parse first, to get a list of sources
             spec = get_rpm_spec(content, rpm.RPMSPEC_ANYARCH | rpm.RPMSPEC_FORCE)
             sources = [s for s, _, _ in spec.sources]
+            del spec
```

Comment:
I'd suggest adding a comment since it's quite unusual to see an explicit `del <object>` in Python code and a comment will make it easier to find the reason than a commit message

Reply:
I thought about it, but since it's on multiple places and the explanation is quite long I wasn't sure how to deal with it. I'm thinking about maybe wrapping `rpm.spec` into a class that would handle this in one place?